pkg/importer/dataStream.go

```diff
@@ -320,19 +320,22 @@ func ResizeImage(dest, imageSize string) error {
 	if err != nil {
 		return err
 	}
-	currentImageSizeQuantity := resource.NewQuantity(info.VirtualSize, resource.BinarySI)
-	newImageSizeQuantity := resource.MustParse(imageSize)
-	minSizeQuantity := util.MinQuantity(resource.NewScaledQuantity(util.GetAvailableSpace(dest), 0), &newImageSizeQuantity)
-	if minSizeQuantity.Cmp(newImageSizeQuantity) != 0 {
-		// Available dest space is smaller than the size we want to resize to
-		glog.Warningf("Available space less than requested size, resizing image to available space %s.\n", minSizeQuantity.String())
-	}
-	if currentImageSizeQuantity.Cmp(minSizeQuantity) == 0 {
-		glog.V(1).Infof("No need to resize image. Requested size: %s, Image size: %d.\n", imageSize, info.VirtualSize)
-		return nil
+	if imageSize != "" && info != nil {
```

Comment:
can info be nil here?  It seems that would be an error in the qemu.Info function, the only reason I ask is the alternate return from this block would be incorrect then.

Reply:
no, since err will be != nil if it is, and thus it will never get here.